### ISSUE TITLE
Stress test option

### ISSUE THREAD

**Issue by Wol377** [RameenAbdal/StyleFlow#37]
I noticed the min / max dicts that feed Parameters_widgets can be changed to allow a wider variety of extreme options which the model accepts.
This is great because you can push the limits of the model and I think most people would enjoy observing what the model does when stressed (expression max of 3 is a joy).

> min_dic = {'Gender': 0, 'Glasses': 0, 'Yaw': -55, 'Pitch': -30, 'Baldness': 0, 'Beard': 0.0, 'Age': 0, 'Expression': -1}
> max_dic = {'Gender': 1, 'Glasses': 1, 'Yaw': 55, 'Pitch': 30, 'Baldness': 1, 'Beard': 1, 'Age': 110, 'Expression': 3}

While I am sure certain combinations will cause model collapse, it would be a shame if the average user didn't have these options.

**Comment by RameenAbdal** [comment]
I would agree that some combinations cause model collapse if subjected to extreme edits. These are some safe ranges where a user can edit with relatively high fidelity. Biases in the sampled data are the main issue. But yeah users can experiment with extreme cases.
